ruby-push-notifications.gemspec

```diff
@@ -20,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
Then it might be simpler to just register the dependency as a dev-time dependency, no?
```
spec.add_development_dependency 'activesupport', '~> 4.2'
```

This way, this requirement is for dev only and has no impact on people trying to add the gem to their projects?